lib/TPP/Dialect/Tpp/BufferizableOpInterfaceImpl.cpp

```diff
@@ -278,10 +332,14 @@ static LogicalResult bufferizeTernaryOp(Operation *op, RewriterBase &rewriter,
 }
 
 // Helper function to bufferize ternay operations.
-// All operands bufferize to memory reads.
 static bool bufferizesToMemoryReadTernaryImpl(Operation *op,
                                               OpOperand &opOperand,
                                               const AnalysisState &state) {
+  // If the rhs input operand is zeroFilled, the access is not read/write
```

Comment:
This isn't `rhs` but the third (accumulation) operand. 'lhs` and `'rhs` only make sense for binary ops.